src/Arcus.Scripting.KeyVault/Arcus.Scripting.KeyVault.psm1

```diff
@@ -0,0 +1,22 @@
+<#
+ .Synopsis
+  Return the current access policies present in Azure Key Vault.
+
+ .Description
+  List the current access policies present in Azure Key Vault.
+
+ .Parameter KeyVaultName
+  The name of the Azure Key Vault from which the access policies are to be retrieved.
+
+ .Parameter ResourceGroupName
+  The resource group containing the Azure Key Vault.
+#>
+function Get-KeyVaultAccessPolicies {
+	param(
+	   [parameter(Mandatory = $true)][string] $KeyVaultName,
+	   [parameter(Mandatory = $false)][string] $ResourceGroupName = ""
+	)
+	. $PSScriptRoot\Scripts\Get-AzKeyVaultAccessPolicies.ps1 -keyVaultName $KeyVaultName -resourceGroupName $ResourceGroupName
+}
+
+Export-ModuleMember -Function Get-KeyVaultAccessPolicies
```

Comment:
Is this correct or do we need the Az prefix here as well?

Reply:
Internal and external command-names should be alligned.

Biggest question is, would you refer to the Az-module in the name of the commands? I don't think we'll support an overload using the AzureRm-module?

Reply:
So the file name has been added with the`Az` prefix but both the internal and external commds have not. Is that okay here to resolve this?

Reply:
I would use Az as a reference to Azure in general so that it's clear we are talking about Azure API Management and not API Management in general for example.

I'd take it all the way and include it in file names, but that's just my opinion!

Reply:
Good point, didn't think outside of the "Azure-box" here.

Let's go for consistency and add the `Az`-prefix all over the place: internal command, external command and filename.

Reply:
Ok, did that.